### ISSUE TITLE
NaN on leverage plots

### ISSUE THREAD

**Issue by srhpatel** [bugsnetsoftware/BUGSnet#6]
I am getting "NaN" for pD and DIC fields of the leverage plots produced using the nma.fit() function.

Could this be because there are 0 events in some of the arms? If so, is there a solution?

Thanks

Shaneel

**Comment by audrey-b** [comment]
Yes, this is the cause. If you look at the output of the nma.fit function you can see that some leverages cannot be computed (NaN). Since our implementation of DIC depends on those leverages, it fails in this case. I don't have a solution to this right now. Perhaps monitoring the DIC directly within JAGS would solve the problem (as Dias's book seem to suggest). This would eventually have to be implemented internally within BUGSnet.

**Comment by Rcher007** [comment]
Hi Audrey, 
I'm having the exact same problem with my project. Are there any updated features within BUGSnet that can tackle this now? 
If not, Can I just pick one model based on visualization and Dres and  use the other model for sensitivity analysis? 

Thanks, 
Don

**Comment by audrey-b** [comment]
I don't have an update on this issue, sorry.

**Comment by Rcher007** [comment]
No worries, Thanks!
BUGSnet is my favorite software for BNMA. Keep up the good work!
